ledger/participant-integration-api/src/main/scala/platform/store/backend/common/EventStorageBackendTemplate.scala

```diff
@@ -389,14 +446,19 @@ abstract class EventStorageBackendTemplate(
       filterParams: FilterParams,
   )(connection: Connection): Vector[EventsTable.Entry[Raw.FlatEvent]] = {
     events(
-      columnPrefix = "",
       joinClause = cSQL"",
       additionalAndClause = cSQL"""
             event_sequential_id > ${rangeParams.startExclusive} AND
             event_sequential_id <= ${rangeParams.endInclusive} AND""",
       rowParser = rawFlatEventParser,
-      selectColumns = selectColumnsForFlatTransactions,
       witnessesColumn = "flat_event_witnesses",
+      partitions = List(
+        "participant_events_create" -> selectColumnsForFlatTransactionsCreate,
+        "participant_events_consuming_exercise" -> selectColumnsForFlatTransactionsExercise,
+        "participant_events_non_consuming_exercise" -> selectColumnsForFlatTransactionsExercise,
+        // Note: previously we used divulgence events, however they don't have flat event witnesses and were thus never included anyway
+        //"participant_events_divulgence" -> selectColumnsForFlatTransactionsDivulgence,
```

Comment:
Is this left out on purpose?

Reply:
Yes, the explanation why it was left out is the comment above.

Reply:
Seems about right. You can decide removing this comment now, or in the next iteration this will be changed anyway.